zcash_proofs/examples/bench.rs

```diff
@@ -1,3 +1,27 @@
+extern crate ff;
+extern crate bellman;
+extern crate pairing;
+extern crate rand_core;
+extern crate rand_xorshift;
+extern crate zcash_primitives;
+extern crate zcash_proofs;
+
+use ff::Field;
+use std::time::{Duration, Instant};
+use zcash_primitives::jubjub::{
```

Comment:
I don't understand why we're importing all of this now if we previously weren't

Reply:
Appears to be superfluous imports. Will remove

Reply:
Can we just delete the lines altogether?